src/test/java/core/basesyntax/service/impl/InputDataValidatorImplTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class InputDataValidatorImplTest {
+    private static InputDataValidatorImpl inputDataValidator;
+    private static final List<String> VALID_INPUT = new ArrayList<>();
```

Comment:
```suggestion
    private static final List<String> validInput = new ArrayList<>();
```
якщо клас не іммютабл, потрібно стандартний неймінг використовувати

Reply:
Ok